lib/screens/about_us/tabs/aim.dart

```diff
@@ -0,0 +1,179 @@
+import 'package:flutter/cupertino.dart';
+import 'package:ecellapp/core/res/colors.dart';
+import 'package:ecellapp/core/res/dimens.dart';
+import 'package:ecellapp/core/res/strings.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class AimScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTextStyle.merge(
+      style: GoogleFonts.roboto().copyWith(color: C.primaryUnHighlightedColor),
+      child: screenBody(context),
+    );
+  }
+
+  Widget screenBody(BuildContext context) {
+    double width = MediaQuery.of(context).size.width;
+    double height = MediaQuery.of(context).size.height;
+    double bottom = MediaQuery.of(context).viewInsets.bottom;
+    double heightFactor = height / 1000;
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Padding(
+          padding: const EdgeInsets.fromLTRB(D.horizontalPadding, 0, D.horizontalPadding, 0),
```

Comment:
add a top padding of `top` (MediaQuery.of(context).viewInsets.top) `+ 56` (height of appbar)